started refining network config chapter (#641)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_retrieving-network-information-from-your-existing-deployment.adoc

    internal_api_subnet:
      ip_subnet: '172.17.0.0/24'
      allocation_pools: [{'start': '172.17.0.4', 'end': '172.17.0.250'}]
----

> . In the `network_data.yaml` file, make a note of the VLAN tag that is used in the `vlan` key and the IP range in the `ip_subnet` key for each isolated network. The IP range is later split into separate pools for control plane services and load balancer IP addresses.

Reviewer: ```suggestion
. Retrieve the VLAN tag that is used in the `vlan` key and the IP range in the `ip_subnet` key for each isolated network from the `network_data.yaml` file. The IP range is later split into separate pools for control plane services and load balancer IP addresses.
```
